src/ceph-volume/ceph_volume/tests/devices/lvm/test_prepare.py

```diff
@@ -102,6 +103,17 @@ def test_journal_is_required_with_filestore(self, is_root, monkeypatch, device_i
         expected = '--journal is required when using --filestore'
         assert expected in str(error.value)
 
+    @patch('ceph_volume.devices.lvm.prepare.api.is_ceph_device')
```

Comment:
You need to patch `ceph_volume.devices.lvm.prepare.get_lv` as well. Otherwise the unit tests try to call an lvm command and some jenkins workers don't have lvm installed and the test will fail.

Reply:
My mistake...this is actually replaced with a Mock.